src/library/scala/util/Random.scala

```diff
@@ -43,11 +44,27 @@ class Random(val self: java.util.Random) extends AnyRef with Serializable {
    */
   def nextDouble(): Double = self.nextDouble()
 
+  /** Returns the next pseudorandom, uniformly distributed double value
+   *  between min (inclusive) and max (exclusive) from this random number generator's sequence.
+   */
+  def between(minInclusive: Double, maxExclusive: Double): Double = ThreadLocalRandom.current().nextDouble(minInclusive, maxExclusive)
```

Comment:
umm, _**making it impossible to get reproducible sequences of value**_ is this for testing purpose ?
To me it looks reasonable to maintain backward compatibility considering the little effort required.

Speaking of testing, can't one always make a proxy object ?